dockerfiles/hpcc/values.schema.json

```diff
@@ -246,6 +261,44 @@
         },
         "image": {
           "$ref": "#/definitions/image"
+        },
+        "logging": {
+          "type": "object"
+        }
+      }
+    },
+	"logging": {
+      "type": "object",
+      "properties": {
+        "fields": {
+          "type": "string",
+          "description": "List of fields to include in logging output. Comprised of 3 letter codes delimited by + or -"
+        },
+        "audiences": {
+          "type": "string",
+          "description": "List of target audiences to include in logging output. Comprised of 3 letter codes delimited by + or -"
+        },
+        "classes": {
+          "type": "string",
+          "description": "List of target logging classes to include in output. Comprised of 3 letter codes delimited by + or -"
+        },
+        "detail": {
+          "type": "integer",
+		  "description": "Log output verbosity"
+        },
+		"useLogQueue": {
+          "type": "boolean"
+        },
+		"queueLen": {
+          "type": "integer",
+		  "description": "Ignored if 'useLogQueue' is false"
```

Comment:
This doesn't tell you what the property means

Reply:
Once I get a description of the field's function, I could fill in the description. Same for below field.